controllers/postController.js

```diff
@@ -0,0 +1,43 @@
+const bodyParser    = require('body-parser');
+const mongoose      = require('mongoose');
+const Post          = require('../models/Post');
+const PostService   = require('../services/PostService');
+const express       = require('express');
+
+const router        = express.Router();
+const urlencodedParser = bodyParser.urlencoded({ extended: false });
+
+const postService = new PostService();
+
+router.get(['/', '/posts'], (req, res) => {
+    postService.getAllPosts().then(posts => {
+        res.render('posts', {posts: posts});
+    }).catch((err) => {
+        res.status(503).send(err.message)
+    });
+});
+
+router.get('/posts/:post_id', (req, res) => {
+    postService.getOnePost(req.params.post_id).then((post) => {
```

Comment:
Consistency with above; either use parenthesis or not for arrow functions with a single argument
eg: `a =>` or `(a) =>`, but be consistent